docs/open-source-project/release-notes/tizen-4-0-m1.md

```diff
@@ -154,7 +154,7 @@ The Tizen 4.0 Public M1 release provides developers with the Tizen kernel, devic
 - Tizen HAL
   - Tizen display HAL
     - Support has been added for the layer commit to ensure an output commit per each vblank:
-      - `TDM_COMMIT_PER_VBLANK` environment
+      - `TDM_COMMIT_PER_VBLANK` environment.
```

Comment:
It looks like this line is not a "sentence". If so, does this line need period?

Reply:
I have removed the period